_pages/en/target-solution-delivery-model-strategy-areas-of-concerns.md

```diff
@@ -30,353 +32,293 @@ This document will be updated throughout the elaboration of the above strategy.
+Over the years, IITB has spent efforts centralizing certain functions to achieve efficiencies and be able to do more with less.
 
-- BSIM
+In the case of the Data Analytics team, direct access to a database as opposed to an API (exposing data meant for consumption by a User Interface or another functional API) is necessary due to the broad nature of analytics and its need for vast amount of data.
+An API may be built, per product, for the sole purpose of accessing the database content but:
 
-#### 2.9.3. What is being done about this concern <!-- omit in toc -->
+1. It's unlikely a Product will have it built, as it will require more costs to do so
+2. Performance hits are expected
+3. The number of APIs to consume and reconstruct data will increase complexity of Data Analytics solutions
 
-TBD.
+#### 2.6.2. Raised by stakeholder(s) <!-- omit in toc -->
 
-Explore multi-gated (3 and 4) PMLC approach.
+- DAS
 
-### 2.10. Risk to capability-based planning ability
+#### 2.6.3. What is being done about this concern <!-- omit in toc -->
 
-#### 2.10.1. Summary <!-- omit in toc -->
+This concern is not addressed at the strategy level (guiding policy) but needs to be reminded during its implementation.
+The following two careful considerations are necessary:
 
-There is a risk that smaller investments promote continuing investing in the wrong IT Product and, as such, limit the organization's ability to improve a given capability.
+1. Data Architecture may be too much of a specialized expertise to realistically be embedded within all DevOps team members (like coding, testing, and securing).
+As such, Data Architects would be individuals with membership on multiple DevOps teams and retain this specialized skill, especially for modelling API schemas and relational databases.
+This consideration regards the [DevOps teams standard definition](https://014gc.sharepoint.com/:w:/r/sites/TargetITSolutionDeliveryModel/_layouts/15/Doc.aspx?sourcedoc=%7BD84FE929-6C0B-469F-B2D4-2D3ECADEBF4F%7D&file=standard-devops-team-definition.docx&action=default&mobileredirect=true)  (one action in the section 3. Coherent set of actions).
 
-A potential scenario is that what was thought of spending $500k to a legacy product one time, comes up as a series of small investments over the year aggregated to more than $2.5M.
-In hindsight, the organization may have preferred to use that $2.5M towards transitioning to a modern IT Product instead.
+2. IT Products must be architected with the notion that large amount of certain data may be required to be consumed by a specialized function (Data Analytics).
+Restful APIs do not provide the necessary performance requirement to transfer such large amount of data.
+Other protocols, such as Secure File Transfer Protocol (sFTP) or some of the [public cloud provider's services designed for such](https://docs.aws.amazon.com/emr/latest/ManagementGuide/emr-plan-input-import-export.html).
```

Comment:
Do you mean using sFTP in combination with a database dump done locally on the server for example? If so I can add more details if you want. 

And regarding the link to AWS, it's for their import/export service of sending a physical storage device. Not sure if it's the right service to include as an example. Maybe linking to an advanced database-as-a-service export feature would be better?

Reply:
@gabrielcossette yes (sFTP with database dump). Only as an example. The idea is to demonstrate that APIs (e.g. Restful APIs) are not always the best of options. And Analytics deal with an enormous amount of data so we need to be mindful of that.

I don't know enough about database-as-a-service export features to explain. But I do recall reading that in some cases, a physical means of transmitting data (even via bus/truck) is faster than over the wire. If you have more examples that can better represent the situation, please update the doc :)

Reply:
I made that part more generic and removed the sFTP example as I'm not sure if it's necessary in that new version. Feel free to readd and tweak:

> Options may be to export the database in the cloud environment, and analyze its content using cloud tools. Or to analyze locally, we can download the export through electronic transfer, or even actual physical transport of storage devices for very large amount of data (e.g., example of service from [AWS](https://docs.aws.amazon.com/emr/latest/ManagementGuide/emr-plan-input-import-export.html)).